src/org/gk/pro/ProExporter.java

```diff
@@ -0,0 +1,643 @@
+        String[] attributes = {ReactomeJavaConstants.compartment,
+                               ReactomeJavaConstants.endCoordinate,
+                               ReactomeJavaConstants.referenceEntity,
+                               ReactomeJavaConstants.stableIdentifier,
+                               ReactomeJavaConstants.startCoordinate};
+        dba.loadInstanceAttributeValues(humanEwasCollection, attributes);
+
+        return humanEwasCollection;
+    }
+
+    private String getRow(GKInstance ewas, List<Object> modifiedResidues) throws InvalidAttributeException, Exception {
+        List<String> row = new ArrayList<String>();
+        String output = "";
+
+        // Entity type (Protein, Complex, etc).
+        final String entityType = ProExporterConstants.ewas;;
+        row.add(entityType);
+
+        // Reactome identifier (R-HSA-).
+        String identifier = getIdentifier(ewas);
+        row.add(identifier);
+
+        // Subcellular location (GO:).
+        String location = getLocation(ewas);
+        row.add(location == null ? ProExporterConstants.unknown : location);
+
+        // UniProtKB accession (with specific isoform, if indicated).
+        String accession = getUniprotAccession(ewas);
+        row.add(accession);
+
+        // Start position of the sequence (if unknown, use '?').
+        Integer startPosition = getStartPosition(ewas);
+        row.add(startPosition == null ? ProExporterConstants.unknown : String.valueOf(startPosition));
+
+        // End position of the sequence (if unknown, use '?').
+        Integer endPosition = getEndPosition(ewas);
+        row.add(endPosition == null ? ProExporterConstants.unknown : String.valueOf(endPosition));
+
+        // Modifications.
+        String modifications = getModifications(modifiedResidues);
+        row.add(modifications == null ? "" : modifications);
+
+        // Free text (where necessary),
+        String freeText = getFreeText(modifiedResidues);
+        row.add(freeText == null ? "" : freeText);
+
+        // Display Name
+        String displayName = ewas.getDisplayName();
+        row.add(displayName);
+
+        output += row.get(0);
+        for (String cell : row.subList(1, row.size()))
+            output += ProExporterConstants.delimiter + cell;
+
+        return output;
+   }
+
+    @SuppressWarnings("serial")
+    private static class MissingPropertyException extends IllegalStateException {
+        public MissingPropertyException(String errorMsg) {
+            super(errorMsg);
+        }
+    }
+
+    /**
+     * @param args
+     * @throws Exception
+     */
+    public static void main(String[] args) throws Exception {
+        ProExporter exporter = new ProExporter();
+        // Get property file.
+        String propPath = null;
+        if (args.length == 0) {
+            // Default property file.
+            final String propertyFile = "proExport.prop";
+            URL pathURL = ProExporter.class.getResource(propertyFile);
```

Comment:
What is the location you expect the user to  place this configuration file? There is no need to do this check. Make it simpler.

Reply:
Simplified! The property file is not expected at the root directory of the project (similar to property files in `gkdev`).